### DISCUSSION TITLE
How to view results,  with ParaView?

### DISCUSSION THREAD

**Discussion by dtlinker** [nextfoam/baram#131] [General]
I have finally successfully run the tutorial "Axi-symmetric nozzle" but can't figure out how to view the results. I assume I should use paraView, but what file to open with ParaView to see the beautiful illustration at the beginning of the tutorial, or any results, for that matter?

**Comment by dtlinker** [comment]
Answering my own question:
It is necessary to navigate to the folder named 'case' and open the file called 'case.foam'. It will look like nothing has happened in ParaView, but in the tile called 'Pipeline Browser' the name 'case.foam' will be displayed. In the tile 'Propoerties' there is a button called 'Apply'. Press that button and the load will be complete.

I have still not figured out how to display the different parts of the dataset, but I have at least answered the original question.

**Comment by kdg5424** [comment]
Hello @dtlinker. This is Donggyu.
Sorry for the late reply.
Could you check which 'Case Type' configure?
If you use just 1core, you should select 'Reconstructed Case'.
But if you use multi-cores, you should selec 'Decomposed Case'.
![image](https://github.com/user-attachments/assets/d4170608-5f1a-434d-88cf-59fffc8e8fd0)

Best regards

Donggyu Kim
